File: Cosmos-Android/app/src/main/java/wannabit/io/cosmostaion/model/kava/DebtParam.java
package wannabit.io.cosmostaion.model.kava;

import com.google.gson.annotations.SerializedName;

public class DebtParam {
    @SerializedName("denom")
    public String denom;

    @SerializedName("reference_asset")
    public String reference_asset;

    @SerializedName("conversion_factor")
    public String conversion_factor;

    @SerializedName("debt_floor")
    public String debt_floor;

    @SerializedName("savings_rate")
    public String savings_rate;
}

File: Cosmos-IOS/Cosmostation/Cosmostation/Model/KavaModel/DebtParam.swift
//
//  DebtParam.swift
//  Cosmostation
//
//  Created by 정용주 on 2021/03/04.
//  Copyright © 2021 wannabit. All rights reserved.
//

import Foundation

public struct DebtParam {
    var denom: String?
    var reference_asset: String?
    var conversion_factor: String?
    var debt_floor: String?
    var savings_rate: String?
    
    init(_ dictionary: NSDictionary?) {
        self.denom = dictionary?["denom"] as? String
        self.reference_asset = dictionary?["reference_asset"] as? String
        self.conversion_factor = dictionary?["conversion_factor"] as? String
        self.debt_floor = dictionary?["debt_floor"] as? String
        self.savings_rate = dictionary?["savings_rate"] as? String
    }
}
